src/concepts/glean/glean.md

```diff
@@ -1,46 +1,69 @@
-# Glean - product analytics & telemetry
+# Glean
 
-For Mozilla, getting reliable data from our products is critical to inform our decision making. Glean is our new product analytics & telemetry solution that provides that data for our products.
-It aims to be easy to integrate, reliable and transparent by providing an SDK and integrated tools.
+For Mozilla, getting reliable data from our products is critical to inform our decision making. Glean is our new product analytics & telemetry solution that will provide consistent data behavior and development experience across all of our products.
```

Comment:
nit: Is the use of the future tense intended here? Does this mean we're not "consistent" yet?

Reply:
Ha.  Just missed updating that.  A lot of the original text was pretty out of date.